cves/CVE-2018-8014.yml

```diff
@@ -187,8 +214,13 @@ lessons:
     applies:
     note:
   secure_by_default:
-    applies:
-    note:
+    applies: true
+    note: |
+      This applies to secure by default, as this issue was not directly addressed
+      in the code until it was found to be an issue. It was assumed that the default 
+      values would have been configured differently by user and there wouldn't be a 
+      problem. This leaves a lesson to be learned by showing that it should not be 
+      assumed that the correct order of steps anticipated will happen every time.
```

Comment:
A brief list of the steps would give slightly more detailed. Other than that i like this

Reply:
I made a brief change to make this more clear.